File: sample/src/main/java/contacts/sample/cheatsheet/other/java/LogContactsApiActivity.java
package contacts.sample.cheatsheet.other.java;

import android.app.Activity;

import contacts.core.*;
import contacts.core.entities.custom.CustomDataRegistry;
import contacts.core.log.AndroidLogger;

public class LogContactsApiActivity extends Activity {

    Contacts createContactsApiWithLoggingEnabled(Boolean redactLogMessages) {
        return ContactsFactory.create(
                this, false, new CustomDataRegistry(), new AndroidLogger(redactLogMessages)
        );
    }
}

File: sample/src/main/java/contacts/sample/cheatsheet/other/kotlin/LogContactsApiActivity.kt
package contacts.sample.cheatsheet.other.kotlin

import android.app.Activity
import contacts.core.Contacts
import contacts.core.log.AndroidLogger

class LogContactsApiActivity : Activity() {

    fun createContactsApiWithLoggingEnabled(redactLogMessages: Boolean): Contacts = Contacts(
        this, logger = AndroidLogger(redactMessages = redactLogMessages)
    )
}